strings.js

```diff
@@ -0,0 +1,263 @@
+// # String Exercises
```

Comment:
Overall good demonstration of string methods and properties. I see some instances of hard coding as changing input wouldn't give us consistent output. Try to create codes that can be flexible and accommodate for differing cases. Keep practicing and keep up the good work. Good job with solving some of bonus questions!